### PR TITLE
Map block stuff

### PR DESCRIPTION
**Pull request by apple502j**
Maps blocks and renames method_10270 (as it's not staticmethod)

### CODE REVIEW

File: mappings/net/minecraft/block/InfestedBlock.mapping [hunk 1]
```diff
@@ -1,6 +1,8 @@
 CLASS net/minecraft/class_2384 net/minecraft/block/InfestedBlock
 	FIELD field_11178 regularBlock Lnet/minecraft/class_2248;
 	FIELD field_11179 REGULAR_TO_INFESTED Ljava/util/Map;
+	FIELD field_33564 REGULAR_STATE_TO_INFESTED_STATE Ljava/util/Map;
+	FIELD field_33565 INFESTED_STATE_TO_REGULAR_STATE Ljava/util/Map;
```
Comment (liach) [inline mappings/net/minecraft/block/InfestedBlock.mapping]:
```suggestion
	FIELD field_33564 REGULAR_TO_INFESTED_STATES Ljava/util/Map;
	FIELD field_33565 INFESTED_TO_REGULAR_STATES Ljava/util/Map;
```

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping [hunk 2]
```diff
@@ -11,6 +11,7 @@ CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeehiveBlockEntity
 	FIELD field_31313 MIN_OCCUPATION_TICKS_WITHOUT_NECTAR I
 	FIELD field_31314 ANGERED_CANNOT_ENTER_HIVE_TICKS I
 	FIELD field_31315 MIN_OCCUPATION_TICKS_WITH_NECTAR I
+	FIELD field_33570 KEY_TO_REMOVE Ljava/util/List;
```
Comment (enbrain) [inline mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]:
```suggestion
	FIELD field_33570 KEYS_TO_REMOVE Ljava/util/List;
```

**Review by enbrain — APPROVED** [review]

File: mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping (hunk 2 above)
Comment (enbrain) [reply mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]: It would be better if we can describe why they should be removed though.

Comment (liach) [reply mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]: Are these item-form incompatible block entity tag keys? Then `ITEM_INCOMPATIBLE_KEYS` or some better name (yeah, I think my suggested name is not fantastic) following that idea would be more descriptive.

Comment (apple502j) [reply mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]: Wait, I forgot whose keys they are... Thought it was for BeeEntity, lemme see

Comment (apple502j) [reply mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]: Yep I was right, it's for BeeEntity, and removes NBTs on `releaseBee`. Think they are ones that should not be kept when bees go inside hive

Comment (liach) [reply mappings/net/minecraft/block/entity/BeehiveBlockEntity.mapping]:
```suggestion
	FIELD field_33570 BEE_KEYS_TO_REMOVE Ljava/util/List;
```
Guess this?

**Review by enbrain — APPROVED** [review]

**Comment by liach** [comment]
To Player: this and #2376 has conflicts. Feel free to decide which one to merge first and which one to merge after fixing conflict

**Comment by Player3324** [comment]
I went with the larger one first, so please fix this one

**Comment by apple502j** [comment]
Closing; #2376 pretty much includes everything this maps

**Pull request closed without merging**